src/components/TextInputButtonSize/TextInputButtonSize.stories.mdx

```diff
@@ -0,0 +1,92 @@
+import { useState } from 'react';
+import { action } from '@storybook/addon-actions';
+import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs/blocks';
+import Box from '../Box/Box';
+import Button from '../Button/Button';
+import TextInput from '../TextInput/TextInput';
+
+
+<Meta
+  title="Patterns/Text Input : Button Size"
+/>
+
+## Text Input Button Size
+
+Set the size of the text input to `sm`, `md` or `lg`. `md` is the default size.
+
+<Canvas>
+  <Story name="TextInputButtonSize">
+    {() => {
+      const [value, setValue] = useState('');
+      const [value1, setValue1] = useState('');
+      const [value2, setValue2] = useState('');
+      return (
+        <div>
```

Comment:
```suggestion
        <Box childGap="sm">
```

Use a `Box` with `childGap` to add space between the size combinations so they aren't bunched together.

<img width="482" alt="Screen Shot 2020-10-26 at 5 47 40 PM" src="https://user-images.githubusercontent.com/1447339/97243341-59510980-17b3-11eb-89f3-97920a0ca8ef.png">

Reply:
do you think we should adjust the radius of the small button to match the input?

Reply:
the height of the large input is 55px while the height of the Large Button is 52px. they should be the same

Reply:
I have Fixed these.